### PR TITLE
Problem: tracer config is not aligned with go-ethereum

### PR DESCRIPTION
**Pull request by mmsqe**
for more [info](https://github.com/ethereum/go-ethereum/blob/v1.11.2/eth/tracers/native/call.go#L109)

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (mempool) [#310](https://github.com/crypto-org-chain/ethermint/pull/310) disable vesting messages in check tx mode.
 * (rpc) [#364](https://github.com/crypto-org-chain/ethermint/pull/364) Only use NextBaseFee as last item to avoid concurrent write in `eth_feeHistory`.
 * (config) [#365](https://github.com/crypto-org-chain/ethermint/pull/365) Avoid redundant parse chainID from gensis when start server.
+* (rpc) [#](https://github.com/crypto-org-chain/ethermint/pull/) Align tracer config with go-ethereum.
```
Comment (mmsqe) [inline CHANGELOG.md]:
```suggestion
* (rpc) [#382](https://github.com/crypto-org-chain/ethermint/pull/382) Align tracer config with go-ethereum.
```

File: rpc/namespaces/ethereum/debug/api.go [hunk 2]
```diff
@@ -76,14 +76,14 @@ func NewAPI(
 
 // TraceTransaction returns the structured logs created during the execution of EVM
 // and returns them as a JSON object.
-func (a *API) TraceTransaction(hash common.Hash, config *evmtypes.TraceConfig) (interface{}, error) {
+func (a *API) TraceTransaction(hash common.Hash, config *rpctypes.TraceConfig) (interface{}, error) {
```
Comment (yihuang) [inline rpc/namespaces/ethereum/debug/api.go:79]: the rpc interface is still not aligned with go-ethereum, right?

Comment (yihuang) [reply rpc/namespaces/ethereum/debug/api.go:79]: how about reuse the [`TraceConfig`](https://github.com/ethereum/go-ethereum/blob/v1.11.3/eth/tracers/api.go#L176) defined in go-ethereum directly?

Comment (mmsqe) [reply rpc/namespaces/ethereum/debug/api.go:79]: not sure, ethermint one contains more field like [limit](https://github.com/crypto-org-chain/ethermint/blob/develop/x/evm/types/trace_config.pb.go#L42)

**Review by yihuang — APPROVED** [review]

**Pull request merged**